src/components/Cards/Cards.component.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react';
+import Card from '../../components/Card';
+
+function Cards(props) {
+  return (
+    <Fragment>
+      {props.videos.items.map((video) => {
+        if (video.id.kind === 'youtube#video') {
```

Comment:
this could be cleaner having this `if` statement as a filter outside of the return

something like
```
const filteredVideos = props.videos.items.filter(video => video.id.kind === 'youtube#video');
``` 

and then you can map over `filteredVideos`

```
filteredVideos.map(video => {
...
}
```